web/src/features/Dashboard/api/getUpdateDownloadStatus.ts

```diff
@@ -28,32 +22,27 @@ const getUpdateDownloadStatus = async (
   );
 
   if (!res.ok) {
-    throw new Error("Error getting update status");
+    throw new Error("failed to get rewrite status");
   }
   const appResponse = await res.json();
-  return appResponse;
-};
 
-const makeUpdateStatusResponse = (
-  response: UpdateStatusResponse
```

Comment:
removing this since checkingForUpdateError, checkingForUpdates, checkingUpdateMessage is set by other functions in Dashboard.tsx. 
We will update state in onSuccess function.